ci/bootstrap_create_accounts.py

```diff
@@ -74,6 +74,7 @@ async def main():
     users = [
         # username, login_id, is_developer, is_service_account
         ('auth', None, 0, 1),
+        ('batch', None, 0, 1),
```

Comment:
Looks like batch was added manually in production? Feels like a mistake that it wasn't in here, and this is necessary for batch to use its hail credentials within the system.